recipes/python-pushover/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
You should remove this file since the license file is already in the source distribution